execution/executor/src/components/apply_chunk_output.rs

```diff
@@ -348,13 +348,13 @@ impl ApplyChunkOutput {
                 txn_info,
                 state_updates,
                 write_set,
-                events,
+                events.clone(),
```

Comment:
can you move events here, and only clone what you need for notification ?

Reply:
fixed